lib/spack/spack/cmd/ci.py

```diff
@@ -119,6 +120,9 @@ def setup_parser(subparser):
     )
     generate.set_defaults(func=ci_generate)
 
+    spack.cmd.common.arguments.add_concretizer_args(generate)
+    spack.cmd.common.arguments.add_common_arguments(generate, ["jobs"])
```

Comment:
For my education, can you explain how this `-j` value ends up affecting the number of processes used for concretization?  I see how you've used it for `rebuild`, just wondering why something similar isn't necessary for `generate`.

Reply:
so, concretization uses the `build_jobs` config parameter to decide how many concretization tasks to spawn for concretize seperately.

The call stack is `lib/spack/spack/concretize.py` calls `spack.config.determine_number_of_jobs` and then queries the config for `config:build_jobs` or a default of 16.

Reply:
How does `config:build_jobs` get set?